docs/UserGuide.md

```diff
@@ -157,6 +159,25 @@ Format: `delete INDEX`
 Examples:
 * `list` followed by `delete 2` deletes the 2nd application.
 
+### Finding application(s) : `find`
+
+Find existing applications in SoC InternApply.
+
+Format: `find [n/NAME] or find [j/JOB_TITLE] or find [t/TAG]... or find [pt/PRIORITY_TAG] or find [ast/APPLICATION_STATUS_TAG]`
+- Finds the applications with the fields containing keywords given in the input.
+- Keywords are case-insensitive.
+- Parameters are only expected once (except tags). e.g. `find n/shopee n/grab` is equivalent to `find n/grab`, the last occurrence of the parameter will be taken.
```

Comment:
I like how you provided the equivalent command for multiple parameters and multiple fields.